tag-and-push.sh

```diff
@@ -15,8 +15,9 @@ function main() {
   # Push debian tags
   if [[ "${target:0:11}" = "debian-base" ]]; then
     flavor=${target:12:$$}
-    # TODO set buster as the primary version for `latest` and tag-based builds
-    primary_debian_version="stretch"
+    # Since this is a 'based images' repo, does it make sense to even provide
+    # a 'default' version? Perhaps all versions should be debian specific?
```

Comment:
Does this need to be addressed for follow-up?

Reply:
In docker-pi-hole the default is already set to specific Debian versions (buster currently). I know of anything using the 'latest' tag. https://github.com/pi-hole/docker-pi-hole/blob/20cd9c3abf95e00d2d1920ba432e2a0272c42942/build.yml#L20